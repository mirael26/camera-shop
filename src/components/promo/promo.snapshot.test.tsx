import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import { promoMock } from '../../test/mocks';
import Promo from './promo';

test('Promo renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<Promo/>, { initialState: { 
    data: { promo: promoMock }
  }});
  expect(asFragment()).toMatchSnapshot();
});
