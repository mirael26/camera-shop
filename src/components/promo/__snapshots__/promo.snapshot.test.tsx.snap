// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Promo renders correctly 1`] = `
<DocumentFragment>
  <div
    class="banner"
    data-testid="banner"
  >
    <picture>
      <source
        srcset="img/content/promo.webp, img/content/promo.webp 2x"
        type="image/webp"
      />
      <img
        alt="баннер"
        height="280"
        src="img/content/promo.jpg"
        srcset="img/content/promo@2x.jpg 2x"
        width="1280"
      />
    </picture>
    <p
      class="banner__info"
    >
      <span
        class="banner__message"
      >
        Новинка!
      </span>
      <span
        class="title title--h1"
        data-testid="promo-title"
      >
        Look 54
      </span>
      <span
        class="banner__text"
      >
        Профессиональная камера от известного производителя
      </span>
      <a
        class="btn"
        href="/catalog/product/7"
      >
        Подробнее
      </a>
    </p>
  </div>
</DocumentFragment>
`;
